docs/modules/ROOT/pages/index.adoc

```diff
@@ -198,7 +201,8 @@ Let us create the component first:
 
 [source,bash]
 ----
-$ commodore component new fortune --owner ${GITHUB_USERNAME}
+$ mkdir -p dependencies/fortune
+$ commodore component new fortune --output-dir dependencies/fortune --owner ${GITHUB_USERNAME}
```

Comment:
This seems unnecessary: `commodore component new fortune --owner ${GITHUB_USERNAME}` automatically creates `dependencies/fortune` (at least for me locally).

Reply:
You are right, fixed with https://github.com/projectsyn/commodore-component-tutorial/pull/14/commits/c52118548427d09bc8138aa6b0f6fa332185e845